test/functional/editions_controller_test.rb

```diff
@@ -64,4 +67,50 @@ class EditionsControllerTest < ActionController::TestCase
       assert_equal EditionsController.new.method(:metadata).super_method.name, :show
     end
   end
+
+  context "when 'restrict_access_by_org' feature toggle is disabled" do
+    %i[show metadata history admin linking unpublish].each do |action|
+      context "##{action}" do
+        setup do
+          @edition = FactoryBot.create(:edition, owning_org_slugs: %w[org-two])
+        end
+
+        should "return a 200 when requesting an edition owned by a different organisation" do
+          login_as(FactoryBot.create(:user, organisation_slug: "org-one"))
+
+          get action, params: { id: @edition.id }
+
+          assert_response :ok
+        end
+      end
+    end
+  end
+
+  context "when 'restrict_access_by_org' feature toggle is enabled" do
+    setup do
+      test_strategy = Flipflop::FeatureSet.current.test!
+      test_strategy.switch!(:restrict_access_by_org, true)
+    end
+
+    teardown do
```

Comment:
more of a question because I am not sure, do we need this teardown block? will the setup in test strategy be enough to make sure switch is set to 'false' before each test?

Reply:
It could affect other tests if we left it turned on (in other test files), unless we turn it off in every other test file. I think it's best turning it off here to make sure.